src/almanac/garden-lattice/garden-lattice.md

```diff
@@ -0,0 +1,32 @@
+# Garden Lattice
```

Comment:
Looks like this duplicates #33

Reply:
Thanks, I'll update this PR before merge with changes from #33.